microcosm_flask/context.py

```diff
@@ -1,23 +1,29 @@
 from flask import request
 from microcosm.api import defaults
+from microcosm.tracing import TRACE_ID, SPAN_NAME
```

Comment:
other services know about support for `tracing` but they don't know anything `jaeger` specific.